core/modules/layout/layout.module

```diff
@@ -2481,6 +2487,20 @@ function layout_layout_info() {
   return $layouts;
 }
 
+/**
+ * Implements deprecated hook_layout_info().
+ *
+ * This hook has been replaced by hook_layout_template_info.
+ *
+ * @deprecated since 1.28.0
+ * @see layout_layout_templatet_info()
+ * @see https://docs.backdropcms.org/node/***** @todo
```

Comment:
```suggestion
 * @deprecated since 1.30.0
 * @see layout_layout_templatet_info()
 * @see https://docs.backdropcms.org/change-records/hook_layout_info-renamed-to-hook_layout_template_info
```